ironfish/src/rpc/routes/chain/isValidPublicAddress.ts

```diff
@@ -0,0 +1,38 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { isValidPublicAddress } from '@ironfish/rust-nodejs'
+import * as yup from 'yup'
+import { ApiNamespace, router } from '../router'
+
+export type IsValidPublicAddressRequest = {
+  address: string
+}
+
+export type IsValidPublicAddressResponse = {
+  valid: boolean
+}
+
+export const IsValidPublicAddressRequestSchema: yup.ObjectSchema<IsValidPublicAddressRequest> =
+  yup
+    .object({
+      address: yup.string().defined(),
+    })
+    .defined()
+
+export const IsValidPublicAddressResponseSchema: yup.ObjectSchema<IsValidPublicAddressResponse> =
+  yup
+    .object({
+      valid: yup.boolean().defined(),
+    })
+    .defined()
+
+router.register<typeof IsValidPublicAddressRequestSchema, IsValidPublicAddressResponse>(
+  `${ApiNamespace.chain}/isValidPublicAddress`,
+  IsValidPublicAddressRequestSchema,
+  (request): void => {
+    request.end({
+      valid: isValidPublicAddress(request.data.address),
```

Comment:
Should we use this `isValidPublicAddress` https://github.com/iron-fish/ironfish/blob/bb99849b03d744454f1098029f836704af94ed7a/ironfish/src/wallet/validator.ts#L12

Reply:
Yeah, it won't practically make a difference, but I think this is a good idea. We should have a cleanup task for standardizing all uses of the function to go through the SDK, and this function is exported through the SDK + used in the wallet

Reply:
Yea sounds good I updated it